src/posit/connect/content.py

```diff
@@ -189,9 +202,7 @@ def dashboard_url(self) -> str:
     def app_role(self) -> str:
         return self.get("app_role")  # type: ignore
 
-    @property
-    def permissions(self) -> Permissions:
-        return Permissions(self.config, self.session, content_guid=self.guid)
+    # Accessor Methods
```

Comment:
Is `delete` an accessor method?

Reply:
Since the list of `@property` methods was getting long, I felt the need to organize it. I've used "accessor" to describe things that access external data sources over the years. But maybe "CRUD Methods" is more suitable here?

 ```suggestion
     # CRUD Methods
```